File: 윤년 (Leap Year)/level01_leapyear.cpp
#include <iostream>
using namespace std;
int main() {
	
	int year;
	cin >> year;
	
	if(year % 4 == 0 || year % 400 == 0){
		if(year % 100 == 0){
			if(year % 400 == 0) cout << "Leap Year";
			else cout << "Not Leap Year";
		} 
		else cout << "Leap Year";
	} 
	else cout << "Not Leap Year";
	return 0;
}

File: 윤년 (Leap Year)/level01_leapyear.java
import java.io.*;
import java.util.Scanner;

class Main {
	public static void main(String[] args) throws Exception {
		Scanner scan = new Scanner(System.in);

		int year = scan.nextInt();
		
		if(year % 4 == 0 || year % 400 == 0){
			if(year % 100 == 0){
				if(year % 400 == 0) System.out.print("Leap Year");
				else System.out.print("Not Leap Year");
		} 
		else System.out.print("Leap Year");
		} 
		else System.out.print("Not Leap Year");
	
	}
}